scripts/pasha_pagerduty.coffee

```diff
@@ -200,11 +294,20 @@ module.exports = (robot) ->
         response = "#{botName} alert trigger <service_name> <description>: " +
             "triggers an alert to the service with the specified name\n" +
             "#{botName} alert list: " +
-            "lists the details of the active Pagerduty alerts"
+            "lists the details of the active Pagerduty alerts\n" +
+            "#{botName} alert phone <email>: " +
+            "return the phone number of a user\n"
```

Comment:
the command name here is misleading, for a command like this i would expect that pasha actually does alert the person